add docs meta descriptions (#32771)
hashicorp/terraform: website/docs/cli/cloud/migrating.mdx

---
page_title: Initializing and Migrating to Terraform Cloud - Terraform CLI
description: >-

>   Learn how to migrate from local state, state backends, or a remote backend to Terraform Cloud with the Terraform CLI.

Reviewer: ```suggestion
  Learn how to use the Terraform CLI to migrate local or remote state to Terraform Cloud.
```

if still accurate
